### Eclipse Workspace Patch 1.0
#P org.pathvisio.desktop
Index: src/org/pathvisio/desktop/MainPanelStandalone.java
===================================================================
--- src/org/pathvisio/desktop/MainPanelStandalone.java  (revision 3908)
+++ src/org/pathvisio/desktop/MainPanelStandalone.java	(working copy)
@@ -102,6 +102,8 @@
 		pluginsMenu.add(standaloneActions.newPluginManagerAction);
 		pluginsMenu.add(standaloneActions.loadLocalBundlesAction);
 		
+		JMenu wikipathwaysMenu = new JMenu("Wikipathways");//new
+		
 		JMenu helpMenu = new JMenu("Help");
 		helpMenu.add(actions.aboutAction);
 		helpMenu.add(standaloneActions.helpAction);
@@ -111,6 +113,7 @@
 		mb.add(dataMenu);
 		mb.add(viewMenu);
 		mb.add(pluginsMenu);//new
+		mb.add(wikipathwaysMenu);//new added
 		mb.add(helpMenu);
 	}
 
